### PR TITLE
Feat/user page component

### PR DESCRIPTION
**Pull request by pitb2022**

### CODE REVIEW

File: src/common/styles/variables/colors.css [hunk 1]
```diff
@@ -20,6 +20,7 @@
 
   /* hover */
   --color-matters-gold-active: #b69c66;
+  --color-matters-green-active: #0a524f;
```
Comment (robertu7) [inline src/common/styles/variables/colors.css:23]: Shall we change the `--color-matters-green-dark` to this new color?

File: src/components/UserProfile/AsideUserProfile/styles.module.css [hunk 2]
```diff
@@ -0,0 +1,118 @@
...
+    color: var(--color-matters-green);
+    background-color: var(--color-white);
+    border: 2px solid var(--color-matters-green);
+    border-radius: 50%;
+
+    &:hover,
+    &:focus {
+      color: var(--color-white);
+      background-color: var(--color-matters-green);
+      transition: color 0.3s ease-in-out, background-color 0.3s ease-in-out;
```
Comment (robertu7) [inline src/components/UserProfile/AsideUserProfile/styles.module.css:24]: `transition` should be placed to normal selector (`svg` L19)?

File: src/components/Avatar/styles.module.css [hunk 3]
```diff
@@ -162,6 +162,34 @@
...
 
+/* 76px */
+.xxxlm {
+  width: 4.75rem;
+  height: 4.75rem;
+
+  &.hasBadge {
+    & .badge {
+      & svg {
+        transform: translateX(25%);
```
Comment (robertu7) [inline src/components/Avatar/styles.module.css:173]: I couldn't get this changes XD...

**Review by robertu7 — APPROVED** [review]
Left some comment, overall looks good. 🧵

File: src/components/Avatar/styles.module.css (hunk 3 above)
Comment (pitb2022) [reply src/components/Avatar/styles.module.css:173]: For Logbook?

File: src/components/UserProfile/AsideUserProfile/styles.module.css (hunk 2 above)
Comment (pitb2022) [reply src/components/UserProfile/AsideUserProfile/styles.module.css:24]: Removed

File: src/components/UserProfile/WalletLabel/index.tsx [hunk 4]
```diff
@@ -1,34 +1,48 @@
 import contentHash from '@ensdomains/content-hash'
+import { FormattedMessage } from 'react-intl'
 import { namehash } from 'viem/ens'
 import { useContractRead, useEnsName, useEnsResolver } from 'wagmi'
 
+// import { namehash } from 'viem/ens'
```
[the remark this answers is not in the document]
Comment (pitb2022) [reply src/components/UserProfile/WalletLabel/index.tsx:6]: done

**Pull request merged**